sentry-logback/src/main/java/io/sentry/logback/SentryAppender.java

```diff
@@ -33,6 +34,7 @@
 /** Appender for logback in charge of sending the logged events to a Sentry server. */
 @Open
 public class SentryAppender extends UnsynchronizedAppenderBase<ILoggingEvent> {
+  // WARNING: Do not use these options in here, they are only to be used for startup
```

Comment:
hm, I'm confused again, who's calling `setOptions` on the SentryAppender actually? Feels a bit wrong that we have an options instance for a specific usecase - should the `start` method accept it as a parameter then? Or it's not possible? Sorry, not aware how logback works internally

Reply:
Created https://github.com/getsentry/sentry-java/issues/2053